frontend/src/components/layout/SideNavBar/ResearchTray.tsx

```diff
@@ -0,0 +1,38 @@
+import { Claims } from 'constants/claims';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { ISideTrayPageProps } from './SideTray';
+import * as Styled from './styles';
+
+/**
+ * Research Tray page.
+ * Intended for use within the left side tray.
+ */
+export const ResearchTray = ({ onLinkClick }: ISideTrayPageProps) => {
+  const { hasClaim } = useKeycloakWrapper();
+  return (
+    <>
+      <HalfHeightDiv>
+        <Styled.TrayHeader>Research Files</Styled.TrayHeader>
+        {hasClaim(Claims.LEASE_VIEW) && (
```

Comment:
Hmm, we probably shouldn't be using the lease claims for this.

Reply:
yep, didn't add those for this story. Added it on the create story